WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsView.swift

```diff
@@ -400,17 +408,16 @@ private extension WooShippingCreateLabelsView {
                 Text(Localization.BottomSheet.orderDetails)
                     .footnoteStyle()
             }
-            AdaptiveStack {
+            HStack {
```

Comment:
Nit: setting `top` alignment would make the layout look nicer in this case:

![image](https://github.com/user-attachments/assets/729c827a-031d-4006-85db-8badf2ce4daf)